packages/@storefront/flux-capacitor/src/core/reducers/data/navigations.ts

```diff
@@ -65,21 +67,30 @@ export const receiveNavigations = (state: State, navigations: Store.Navigation[]
 };
 
 // tslint:disable-next-line max-line-length
-export const selectRefinement = (state: State, { navigationId, index: refinementIndex }: Actions.Payload.Navigation.Refinement) => {
-  if (navigationId && refinementIndex != null) {
-    return {
-      ...state,
-      byId: {
-        ...state.byId,
-        [navigationId]: {
-          ...state.byId[navigationId],
-          selected: state.byId[navigationId].selected.concat(refinementIndex),
-        },
+export const selectMultipleRefinements = (state: State, { navigationId, indices }: Actions.Payload.Navigation.MultipleRefinements) => {
+  if (!navigationId || indices.length === 0) {
```

Comment:
Can either do `indices.length` or not check this bc the actions that end up here have validation to ensure they're given an array

Reply:
Will remove, I don't think we need it anymore.